File: programacion actividades/ejerciciosU2/src/EJ1.java

import java.util.Scanner; /*las librerias van siempre antes del class555*/

class Scratch {


    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);  /*para aqui se pone el nombre que quieras, despues para las variables habria que poner "nombre".next"variable"*/
        System.out.println("introduce un numero, vamos a calcular el area de un rectangulo, introduce la base");
        int i = sc.nextInt();
        System.out.println("ahora la altura");
        int j = sc.nextInt();
        int k = i * j;
        System.out.println("el area es " + k);


    }
}

File: EjerciciosPracticar JUNIO1/src/Ev3/Kotlin/Ej1.kt
package Ev3.Kotlin

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)

    println("Introduce dos numeros")
    val a = scanner.nextInt()
    val b = scanner.nextInt()

    if (a > b){
        print("${a} es mayor que ${b}")
    }else{
        print("${b} es mayor que ${a}")

    }
}